### DISCUSSION TITLE
Problem with grass_side.tga?

### DISCUSSION THREAD

**Discussion by erich666** [jasonjgardner/jg-rtx#32] [Feedback]
The new:

![image](https://user-images.githubusercontent.com/803995/117043757-a4f0ec80-acdb-11eb-8065-fdf8908b0005.png)

I would expect the grayscale image of the grass match the color one. The grayscale's now too far down.

This is how it used to be:

![image](https://user-images.githubusercontent.com/803995/117044024-eda8a580-acdb-11eb-849e-5fb7a04f0f5a.png)

so I'll use the old grayscale image for now.

**Comment by jasonjgardner** [comment]
The carried grass texture probably didn't get updated because I decided to do something different with all carried textures around the time I changed the grass texture.

I've started exporting Substance Painter's 2D View as my carried textures.

![Substance Painter export preset](https://user-images.githubusercontent.com/1903667/117459792-66dd0e00-af11-11eb-912a-568b0f1a7219.png)

I can reference the `_carried.png` files in the `blocks.json` file to show the pre-rendered texture in the inventory and while holding a block. These carried textures can then be reused in an eventual RTX-less version of the resource pack.

![Resource pack JSON files](https://user-images.githubusercontent.com/1903667/117462545-642fe800-af14-11eb-8e80-63efd233e910.png)

![Carried textures in action](https://user-images.githubusercontent.com/1903667/117462699-888bc480-af14-11eb-9d69-3f12d1b03135.png)

I realize the carried grass texture has more than one purpose, so I'll quickly fix the mismatched textures.

---

### Is the grass too long?

I'm trying out this longer grass side texture. I wanted to make hills look hillier, but this also gives cliffsides a moptop...

![Long grass](https://user-images.githubusercontent.com/1903667/117464730-917d9580-af16-11eb-8668-97e47b8a6cca.png)

**Reply by erich666** [reply]
I admit to liking the shorter grass, but follow your muse (I can always substitute out if I want the older ones).
